alpyvantage/data_formating.py

```diff
@@ -46,5 +46,11 @@ def _format_to_pandas(call_response, data_key):
     data_pandas.index.name = 'date'
     # convert to pandas._libs.tslibs.timestamps.Timestamp
     data_pandas.index = pd.to_datetime(data_pandas.index)
-    data_pandas.columns = [col.split('. ')[1] for col in data_pandas.columns]
+    data_pandas.sort_index(inplace=True)
```

Comment:
Here is the sort

Reply:
Thanks for the PR! I'm happy to merge. Yes, sorting seems preferable. I'll fix the tests...